### DISCUSSION TITLE
Ideas on the new description feature

### DISCUSSION THREAD

**Discussion by j13m126** [organicmaps/organicmaps#2320] [Ideas]
Let me first say how much I love to see this new feature in OM. I've added wikidata/wikipedia to hundreds of POIs, so it is a great joy and further motivation to see my little contribution in OM. Thank you! :)

I'd like to share my ideas on how it could be improved on Android.

- Clicking on a POI with description available takes up too much space of the screen. Limiting the description to one sentence or a few lines should be enough to show that there is a description. The description panel itself indicates that already. This would also make it more consistent to POIs without a wikipedia description. The issue could also be solved by decreasing the height of the description panel or removing it as it takes up valuable space once you scroll down to the bottom.
 
- The description (once you scroll upwards) should be limited to the actual description, as it's length already varies greatly. The first section of wikipedia is not as important imho. The problem I see here is that all further information shown in OM (like opening hours, url, social media links etc) is buried to the bottom of the screen and is therefore not as easy to discover as before. This of course is only an issue primarily on smaller screens.

Let me know what you think. :)
Thanks!

**Comment by biodranik** [comment]
One of the options would be to show the Wikipedia button as it was before. It will display the detailed descriptions and then can have a link to the original article.

It would be consistent with another feature we want to add: showing an editable "description" tag from the OSM itself.
